pkg/starter/starter.go

```diff
@@ -170,6 +170,9 @@ func (s *Starter) processJob(ctx context.Context, job datastore.Job) error {
 
 		return fmt.Errorf("failed to bung (target ID: %s, job ID: %s): %w", job.TargetID, job.UUID, err)
 	}
+	if resourceType == 0 {
```

Comment:
Please use `Unknown` in package

Reply:
https://github.com/whywaita/myshoes/pull/169/commits/acbbb5ba6437dd4f154bdde2e3a3b1ea044fa550
Fixed in this commit.